### PR TITLE
[92731080] Reformat ssh:sudo message

### PR DESCRIPTION
**Pull request by apotterri**

### CODE REVIEW

File: lib/conjur/command/audit.rb [hunk 1]
```diff
@@ -24,12 +24,22 @@ class << self
...
+        s
+      end
       
       def short_event_format e
         e.symbolize_keys!
         s = "[#{Time.parse(e[:timestamp])}]"
         s << " #{e[:user]}"
         s << " (as #{e[:acting_as]})" if e[:acting_as] != e[:user]
+        if e[:facility] == 'ssh' && e[:action] == 'sudo'
+          e[:audit_message] = ssh_message(e)
```
Comment (hleb-rubanau) [inline lib/conjur/command/audit.rb]: Given the condition above I'd recommend to name the function e.g. "ssh_sudo_message" rather then "ssh_message". There are other messages from ssh facility, describing login and logout, for example, which are not covered by this logic.

But this is rather nitpicking, feel free to ignore it if you want to :)

File: spec/command/audit_spec.rb [hunk 2]
```diff
@@ -312,6 +312,25 @@ def self.it_fails command, *raise_error_args
...
+          let(:test_event) { ssh_event.merge('allowed' => true) }
+          it 'prints <user> ran <command> for successful command' do
+            expect { invoke }.to write(" test_user ran '/bin/ls' as root")
+          end
+        end
+
+        context 'when sudo fails' do
+          let(:test_event) { ssh_event.merge('allowed' => false) }
+          
+          it 'prints <user> attempted to run <command> for successful command' do
```
Comment (dividedmind) [inline spec/command/audit_spec.rb]: `successful`? In general, I think the "for successful command" phrase is redundant with the context.

**Comment by kgilpin** [comment]
@apotterri this is sitting as an open pull; ready to merge?

**Comment by apotterri** [comment]
Yes, I believe so.

On Mon, Jul 27, 2015 at 9:32 AM, Kevin Gilpin <notifications@github.com>
wrote:

> @apotterri <https://github.com/apotterri> this is sitting as an open
> pull; ready to merge?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/conjurinc/cli-ruby/pull/113#issuecomment-125205606>.
>



-- 
*Alan Potter*
Engineer
p: 617.584.7102


 <http://www.conjur.net/>

*Security Orchestration For Next Generation IT*

Demo <http://info.conjur.net/demos> | Resources
<http://info.conjur.net/resources> | Blog <http://blog.conjur.net/>

**Pull request merged**